zcash_client_sqlite/src/error.rs

```diff
@@ -118,17 +120,17 @@ pub enum SqliteClientError {
     /// A note selection query contained an invalid constant or was otherwise not supported.
     NoteFilterInvalid(NoteFilter),
 
-    /// The proposal cannot be constructed until transactions with previously reserved
-    /// ephemeral address outputs have been mined. The parameters are the account UUID and
-    /// the index that could not safely be reserved.
+    /// An address cannot be reserved, or a proposal cannot be constructed until a transaction
+    /// containing outputs belonging to a previously reserved address has been mined. The error
+    /// contains the index that could not safely be reserved.
     #[cfg(feature = "transparent-inputs")]
-    ReachedGapLimit(AccountUuid, u32),
+    ReachedGapLimit(u32, KeyScope),
```

Comment:
Nit: `KeyScope` is a level above the address index, so I'd prefer the tuple being in the opposite order.